cruz_solo_assessment/db/migrate/20150527134253_create_people.rb

```diff
@@ -0,0 +1,23 @@
+class CreatePeople < ActiveRecord::Migration
+  def change
+    create_table :people do |t|
+      t.text :name
+      t.text :title
+      t.text :quote
+      t.text :image
+      t.int :age
+
+      t.timestamps null: false
+    end
+  end
+end
+Daenerys Targaryen = Person.create(name: 'Daenerys Targaryen', title: 'Queen of Meereen and Mother of Dragons', quote: 'I will take what is mine with blood and fire.', image: 'http://i.imgur.com/DjHJq4E.jpg', age: 18
```

Comment:
Get this ish out of your migration file. Put it in either your scripts or your seed file